src/python/bezier/hazmat/algebraic_intersection.py

```diff
@@ -999,7 +999,7 @@ def lu_companion(top_row, value):
     lu_mat = np.zeros((degree, degree), order="F")
     if degree == 1:
         lu_mat[0, 0] = top_row[0] - value
-        return lu_mat, abs(lu_mat[0, 0])
+        return lu_mat, float(abs(lu_mat[0, 0]))
```

Comment:
Arguably this conversion overhead isn't worth it (I did it because the `__repr__` of `numpy.float64` changed in NumPy 2.0.)

Reply:
I elected to do all conversions in `doctest` code, not in source code!